docs/configuration/modules/remote-service.md

```diff
@@ -31,6 +31,28 @@ to connect to the remote server. The default value of this property is `true`.
 The `port` property is an optional property that defines the connection port. By default, if SSL/TLS protocol
 is used, the default value of this property is `443`. Otherwise, it is `80`.
 
+## autoreconnect-enabled
+
+The `autoreconnect-enabled` property is an optional property that defines if the server should be automatically reconnected when the connection is lost. The default value of this property is `false`.
+
+## reconnection-initial-interval
+
+The `reconnection-initial-interval` property is an optional property that defines the initial interval (in seconds) before the first reconnection. Its default values is 5.
```

Comment:
*value
put back-ticks around the default value : *Its default value is `5`

Reply:
Ok It's done